example_configs/zitadel.md

```diff
@@ -0,0 +1,38 @@
+# Configuration for Zitadel
+In Zitadel, go to `Instance > Settings` for instance-wide LDAP setup or `<Organization Name> > Settings` for organization-wide LDAP setup.
+
+## Identity Providers Setup
+Click `Identity Providers` and select `Active Directory/LDAP`.
+
+Replace every instance of `dc=example,dc=com` with your configured domain.
+**Group filter is not supported at the time of writing.**
```

Comment:
Is that an LLDAP limitation, or a Zitadel one?

Reply:
it's zitadel